library/tutorials/Functions - SCORE/0.json

```diff
@@ -187,7 +158,13 @@
     "scoreProperty": "scoreDemo1",
     "movable": false,
     "autosizeColumns": false,
-    "playersInColumns": false
+    "playersInColumns": false,
+    "roundLabel": "",
+    "rounds": [
+      "Score"
+    ],
```

Comment:
That was the source of that suggestion I made during the devlopment of scoreboards. I had a scoreboard that did this. I Think you don't need the [ ] and you only need to point to a property on seats that is a number.

Reply:
True, but if you do that, the width of the score column changes a lot and doesn't present as clean a look.  I'm not sure exactly why, but I'm going to leave it as is.